flexmeasures/data/models/time_series.py

```diff
@@ -72,13 +72,19 @@ def __init__(
     def __acl__(self):
         """
         Within same account, everyone can read and update.
-        Creation and deletion are left to site admins in CLI.
-
-        TODO: needs an iteration
+        Deletion needs the account-admin role.
         """
         return {
+            "create-children": (
+                f"account:{self.generic_asset.account_id}",
+                "role:account-admin",
```

Comment:
Yes. It's a more complicated case. We need to codify that the MDC has some relationship to the account.

This might require that we add such relationships to the `Account` model, or even per Sensor. I'll open a ticket but I see design discussion as the first work. Might lead to a cool USEF-inspired data model!

Reply:
https://github.com/FlexMeasures/flexmeasures/issues/296